pkg/cmd/cli/describe/describer.go

```diff
@@ -118,6 +119,23 @@ func (d *BuildDescriber) DescribeParameters(p buildapi.BuildParameters, out *tab
 	}
 }
 
+func (d *BuildDescriber) DescribeDockerImage(s *buildapi.STIBuildStrategy, namespace string, out *tabwriter.Writer) {
+	if s.Image == "" {
+		c := d.ImageRepositories(namespace)
+		imageRepo, err := c.Get(s.From.Name)
+		if err == nil {
+			formatString(out, "Image Repository", imageRepo.Status.DockerImageRepository)
+		} else {
+			j, _ := json.Marshal(s.From)
+			formatString(out, "Docker Image Error", err)
+			formatString(out, "stisTRATEGY.fROM", string(j))
+			formatString(out, "imageRepo", imageRepo.Status.DockerImageRepository)
```

Comment:
you got an error retrieving the imagerepo, why are you assuming you have a valid imagerepo object here?